frontend/src/store/actions/metrics/getService.ts

```diff
@@ -38,24 +39,24 @@ export const GetService = (
 			selectedTags: props.selectedTags,
 		});
 
-		if (response.statusCode === 200) {
+		if (response.length > 0) {
 			dispatch({
 				type: 'GET_SERVICE_LIST_SUCCESS',
-				payload: response.payload,
+				payload: response,
 			});
 		} else {
 			dispatch({
 				type: 'GET_SERVICE_LIST_ERROR',
 				payload: {
-					errorMessage: response.error || 'Something went wrong',
+					errorMessage: SOMETHING_WENT_WRONG,
```

Comment:
take server error also into account

Reply:
`response.error` was removed as we are not sending an error in the response object from the API call now. It was handled with the help of `error` and `isError` from `useQuery`